.github/CONTRIBUTING.md

```diff
@@ -137,6 +137,7 @@ being accepted. If you are experiencing intermittent TEST
 builder failures, you may be experiencing a [test suite
 issue](https://github.com/zfsonlinux/zfs/issues?q=is%3Aissue+is%3Aopen+label%3A%22Test+Suite%22).
 * All proposed changes must be approved by a ZFS on Linux organization member.
+* All commits require a signed-off-by tag to certify you wrote or otherwise have the right to contribute the changes. For more information, see [here](https://www.kernel.org/doc/html/latest/process/submitting-patches.html#sign-your-work-the-developer-s-certificate-of-origin).
```

Comment:
It may be more appropriate to put this messaging in the commit message format section. It feels like a further explanation as to why the commit message is formatted the way it is.

Also `signed-off-by` should have a capital S and please  surround it with ticks (`).